### ISSUE TITLE
Update the "Data in the Public for the Public" with the version from the designers

### ISSUE THREAD

**Issue by dingaaling** [GLD5000/LDW-GLD#21]
- So that the aesthetic matches the design
- Example SVG: ![Group 16](https://user-images.githubusercontent.com/5104098/232072694-ed00ec34-7dd4-4784-a1c9-1a65053e4af3.svg)
- Example PNG: ![Group 16](https://user-images.githubusercontent.com/5104098/232072703-447b578b-774b-4980-8a45-df4038b97fb6.png)

**Comment by GLD5000** [comment]
Thank you for that- I'm more than happy to go with whatever you decide! 

The reasons why I made it as a non-image, dynamic version rather than using the image in the kit in this first version was because: 
1. Image text is an accessibitiy concern for screen readers
2. Text within images is not accessible for Search Engine Optimisation
3. Images with text are less responsive as they can only be scaled as a whole

In the image below, I have added the image for you to see what it looks like. The image has a fixed aspect ratio of roughly 2:1 (landscape) so appears quite small on mobile phones, and I have already cropped it as much as possible to keep the text centred. 

Let me know what you think. 

![image](https://user-images.githubusercontent.com/33987109/232162354-41dfae15-4fb2-44d3-bcd4-e786d565dc39.png)
